images/node/scripts/origin-node-run.sh

```diff
@@ -2,21 +2,12 @@
 
 set -eu
 
+hostetc=${HOST_ETC:-/rootfs/etc}
 conf=${CONFIG_FILE:-/etc/origin/node/node-config.yaml}
-opts=${OPTIONS:---loglevel=2}
```

Comment:
I can enable options again, although it should probably be an array.

On Mon, Feb 15, 2016 at 3:51 PM, Scott Dodson <notifications@github.com>
wrote:

> In images/node/scripts/origin-node-run.sh
> <https://github.com/openshift/origin/pull/7284#discussion_r52942289>:
>
> >  conf=${CONFIG_FILE:-/etc/origin/node/node-config.yaml}
> > -opts=${OPTIONS:---loglevel=2}
>
> This was how we're passing in items from the systemd environment file
> (/etc/sysconfig/origin-node). I believe this will break that behavior. I'll
> try it as you're proposing and see what happens though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/7284/files#r52942289>.
>